pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -88,6 +91,7 @@ toPythonModule (python.stdenv.mkDerivation (builtins.removeAttrs attrs [
   installCheckInputs = checkInputs;
 
   postFixup = lib.optionalString (!dontWrapPythonPrograms) ''
+    ${if skipNoUserSite then "export SKIPNOUSERSITE=1" else ""}
```

Comment:
A variable such as `skipNoUserSite` is, when passed on, directly an environment variable. `foo = true` will evaluate to foo=1`, and in case of false the variable won't exist. No need to create another one then.

Reply:
Sorry not 100% sure I understand what you mean by "passed on" -- I guess if you pass something like a list or string to `buildInputs` it ends up available as an environment variable? I'm just going to try to mimic the `pythonPath` argument in this file.

Reply:
We call stdenv.mkDerivation and pass it an attribute set. If you simply include the variable in that attribute set (instead of removing it!), then it will end up in the builder as an environment variable.

Reply:
OH I see how this works now, thanks

Reply:
So I should not add it to `buildInputs`, right?

Reply:
nope